### DISCUSSION TITLE
[Feature] Show mod preset's description in the tooltip

### DISCUSSION THREAD

**Discussion by CloneWith** [ppy/osu#28610] [Ideas] [answered]
Like the description text shown in the tooltip of individual mods, the description of a custom mod preset can also be shown when the cursor hovers above the preset in the same way.

This is especially helpful when the description is too long to be displayed in one line.

![图片](https://github.com/ppy/osu/assets/110881926/8616f809-6134-4630-857a-d54c61d1428c)

**Comment by peppy** [comment, marked as answer]
https://github.com/ppy/osu/pull/28650
